### DISCUSSION TITLE
WS doesn't update everything when we try to create a credit slip

### DISCUSSION THREAD

**Discussion by BenTen** [PrestaShop/PrestaShop#33153] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

As #26647, the webservice update only a part of data.

When we try to refund an order or a part of an order and then create a credit slip with the WS, `unit_price` and `total_price` are ignored.

`unit_price` is useful to create the pdf.

And then `product_quantity_refunded` in the table `order_detail` is not updated.

Maybe others are not updated too.

It's result to a buggy pdf

### Expected behavior

The webservice need to help owner to update everything related to credit slip. Not only a part.

### Steps to reproduce

https://devdocs.prestashop-project.org/1.7/webservice/resources/order_slip/

### PrestaShop version(s) where the bug happened

1.7.5.2

### PHP version(s) where the bug happened

7.1

**Comment by aniszr** [comment]
Hello,

Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments.

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you create it.

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop-project.org/8/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

Thank you

**Comment by aniszr** [comment]
Ping @MatShir could you please transfer this issue to discussions :pray: 
thanks in adavance
